themes/custom/droopler_theme/scss/layout/_modifiers.scss

```diff
@@ -15,6 +15,9 @@ $arrow-margin-top: 6% !default;
 $arrow-border: 9px solid transparent !default;
 $arrow-down-color: lighten($color-helen, 10%) !default;
 $stripe-background-color: rgba(0,0,0, 0.57) !default;
+$image-background-padding-y: 110px !default;
```

Comment:
Maybe we should use a name like `tile-padding-top`? But I may not understand the context :)